libafl/src/schedulers/minimizer.rs

```diff
@@ -371,7 +375,8 @@ where
     /// and has a default probability to skip non-faved [`Testcase`]s of [`DEFAULT_SKIP_NON_FAVORED_PROB`].
     /// This will remove the metadata `M` when it is no longer needed, after consumption. This might
     /// for example be a `MapIndexesMetadata`.
-    pub fn new(base: CS) -> Self {
+    pub fn new(_obs: &O, base: CS) -> Self {
```

Comment:
Can we use a full name here? `_obs: O` could be an obstacle or anything :D

Reply:
Also, why have this if it's not used?

Reply:
It's a type hint, because the `O` determines the tracking.